src/display/canvas.js

```diff
@@ -1248,6 +1248,8 @@ class CanvasGraphics {
     this.viewportScale = 1;
     this.outputScaleX = 1;
     this.outputScaleY = 1;
+    this.foregroundColor = pageColors?.foreground || null;
+    this.backgroundColor = pageColors?.background || null;
```

Comment:
Nit: Let's change the order of these two, given the other comments about sorting the properties alphabetically.